Boeing-Ontology/ontology/Boeing.sadl

```diff
@@ -1,20 +1,95 @@
 
 SBVT_Test (note "A test identified in the Software Baseline Verification Tests") is a type of TEST.
 //verifies of SBVT_Test only has values of type {SRS_Req or SubDD_Req}.
 verifies of SBVT_Test has at least 1 value.
 
 SBVT_Result (note "A result identified for a SBVT_Test (Software Baseline Verification Tests)") is a type of TEST_RESULT.
-confirms of SBVT_Result only has values of type SBVT_Test.
\ No newline at end of file
+confirms of SBVT_Result only has values of type SBVT_Test.
+
+////////////////////////////////////// Moved from SRI ontology
+//AM: replaced author by wasAssociatedWith in DevelopSystemConOps
+//AM: dropped author in DevelopSystemArchitecture as we inherit wasAssociatedWith
+//AM: do we want more granularity than just wasAssociatedWith
+
+     DevelopSystemConOps is a type of ACTIVITY.
+       wasAssociatedWith of DevelopSystemConOps only has values of type PERSON.
+       
+     SystemConOps is a type of ENTITY.
+       wasGeneratedBy of SystemConOps only has values of type DevelopSystemConOps.
+       conOpsDocs describes SystemConOps with values of type DOCUMENT.  
+     
+     DevelopSystemArchitecture is a type of ACTIVITY.  
+        wasInformedBy of DevelopSystemArchitecture only has values of type DevelopSystemConOps.
+        used of DevelopSystemArchitecture only has values of type SystemConOps. 
+
+     SystemArchitecture (note "specification of system architecture") is a type of ENTITY. 
+      //AM what is the difference between wasGeneratedBy and original DesCert:createdBy? so using wasGeneratedBy
+       wasGeneratedBy of SystemArchitecture only has values of type DevelopSystemArchitecture. 
+       isArchitectureOf describes SystemArchitecture with values of type SYSTEM.
+       isArchitectureOf is a type of wasImpactedBy.
+
+     SystemRequirementsDefinition is a type of REQUIREMENT_DEVELOPMENT. 
+       wasInformedBy of SystemRequirementsDefinition only has values of type DevelopSystemConOps. 
+     
+      PIDS_Doc is a type of DOCUMENT. 
+       content of PIDS_Doc only has values of type PIDS_Req. 
+       wasGeneratedBy of PIDS_Doc only has values of type SystemRequirementsDefinition. 
+        
+      SoftwareHighLevelRequirementsDefinition is a type of REQUIREMENT_DEVELOPMENT. 
+		wasInformedBy of SoftwareHighLevelRequirementsDefinition only has values of type {SystemRequirementsDefinition or DevelopSystemArchitecture}.
+        Rq:governedBy of SoftwareHighLevelRequirementsDefinition only has values of type {PIDS_Doc or SystemArchitecture}.
+
+//      SoftwareHighLevelRequirement is a type of REQUIREMENT. 
+//        wasGeneratedBy of SoftwareHighLevelRequirement only has values of type SoftwareHighLevelRequirementsDefinition.
+       
+     SRS_Doc is a type of DOCUMENT. 
+       content of SRS_Doc only has values of type SRS_Req. //AM only?
+       createdBy of SRS_Doc only has values of type SoftwareHighLevelRequirementsDefinition.
```

Comment:
`createdBy` is probably going away. putting this constraint on `wasGeneratedBy` will make it relevant in the future